oeps/processes/oep-0060-proc-sec-group.rst

```diff
@@ -0,0 +1,148 @@
+----------------------------------------------------------
+Triage responsibilities include:
+
+* On-call person stays with security issue until it is resolved
+* On-call responsibilities
+
+  * Triage and calculate severity of issue
+  * Notify and work with SME for the impacted area(s)
+  * Get ticket prioritized and resolved (i.e. project manage the ticket(s) in SME backlogs)
+
+    * Maintainers are responsible for resolving security issues
+    * Security Group will set security SLA to manage expectations for time to resolve
+  * Work with SMEs to release the change to master and current supported Open edX release
+
+Because the on-call member is expected to follow a disclosure through its eventual resolution, some of these
+responsibilities will extend beyond the initial scheduled on-call time.
+
+We would also like to specifically note that if an on-call week is particularly time intensive, the on-call member can
+ask the other working group members for support.
+
+.. _security-releases:
+
+Security Releases
+-----------------
+As opposed to the current security process, which involves sending a disclosure and security patch to members of the
+open source security email list and waiting two days before making the patch public, are proposing the following
+security release process:
+
+#. Updates will be merged directly to master and backported to the current supported named release.
+#. At the time that the update is merged to master, the update will also be announced to everyone.
+
+Focus on proactive security improvements
+========================================
+Part of the work for this group should include proactive security improvements to the Open edX codebase, which could
+include the below examples.
+
+Examples
+--------
+#. Security suites as part of Github CI.
+#. Better visibility for security supply chain issues.
+
+   * How can we take advantage of the alerts that Github provides for security prioritization?
+#. Iterations on the security process.
+#. Review industry best practices that we should consider implementing.
+
+Consequences
+************
+
+Security Working Group
+======================
+
+Invite Only
+-----------
+Initial members of the working group will come from the existing internal security group. If more volunteers are needed,
+we will put out a call to join. Volunteers will need to ask for support from other core contributors and will then be
+evaluated by the existing working group members. We hope to have six to seven members of the working group at a time.
+
+Security work will be a mix of private and public tickets
+---------------------------------------------------------
+Proactive work for security improvements will be made public, while vulnerability reports will be handled privately.
+We propose using Github Security Advisories to handle the triage of vulnerability reports to the owners of
+vulernable components.
+
+Security Team Member Responsibilities
```

Comment:
Maybe add something in here about expectations around embargo? That is, that members of the WG (and their employers) agree to keep vulnerabilities private until the coordinated disclosure occurs. This wouldn't preclude early patching unless that would require making the fix public early.

Reply:
I like this, @alangsto let's add it.

Reply:
Updated the responsibilities to include this